app/src/main/java/org/jellyfin/androidtv/util/apiclient/BaseItemDtoExtensions.kt

```diff
@@ -0,0 +1,15 @@
+package org.jellyfin.androidtv.util.apiclient
+
+import android.graphics.Bitmap
+import org.jellyfin.androidtv.model.itemtypes.Episode
+import org.jellyfin.apiclient.model.dto.BaseItemDto
+
+fun BaseItemDto.asEpisode(primaryImage: Bitmap?) : Episode {
```

Comment:
I'm now thinking we should probably just use the image urls in those objects for now. The episode class could contain a function to get the bitmap.

Also, episodes should probably use the "thumb" image instead of primary (I might be wrong about this though)

Reply:
My new idea for this is now to move the download function for images into BaseDetailsFragment and then let each item-specific class do its own handling.

Reply:
Yeah that would work for now